sdk/cosmos/azure-cosmos-spark_3-0_2-12/src/main/scala/com/azure/cosmos/spark/CosmosTable.scala

```diff
@@ -76,6 +76,9 @@ class CosmosTable(val transforms: Array[Transform],
   }
 
   override def capabilities(): util.Set[TableCapability] = Set(
+    // ACCEPT_ANY_SCHEMA is needed because of this bug https://github.com/apache/spark/pull/30273
+    // It was fixed in Spark 3.1.0 but Databricks currently only supports 3.0.1
+    // When Databricks supports 3.1.X, we could remove it and move to a newer Spark version
```

Comment:
Only in new major version - changing capabilities is a breaking change. Leave the first two lines and remove this one and we deal with deciding whether we want to take the new major version later.

Reply:
Done